Examples/MedicationOverview-d44e145.xml

```diff
@@ -0,0 +1,62 @@
+<List xmlns="http://hl7.org/fhir">
```

Comment:
Ik snap de keuze om de id te laten terugkomen in de bestandsnaam, en ook de keuze om het resource-type terug te laten komen in de bestandsnaam, maar daardoor komen id en bestandsnaam niet overeen, wat afwijkt van de rest. Zou het niet mogelijk zijn om juist de resource-type op te nemen in de _id_?